lib/constants.dart

```diff
@@ -42,6 +42,21 @@ abstract class SarakaTextStyles {
     fontSize: 18,
     color: SarakaColors.lightBlack,
   );
+  static final bodySmall = _base.copyWith(
+    fontWeight: FontWeight.w400,
+    fontSize: 12,
+    color: SarakaColors.darkGray,
+  );
```

Comment:
It looks much similar with `body2`. How about use `body2` with `copyWith()` or modify itself (it also affects other parts though)